lib/ts/recipe/emailverification/types.ts

```diff
@@ -0,0 +1,56 @@
+import { NormalisedRecipeConfig, RecipeConfig, RecipeFunctionOptions } from "../recipeModule/types";
+import OverrideableBuilder from "supertokens-js-override";
+
+export type PreAPIHookContext = {
+    action: "VERIFY_EMAIL" | "SEND_VERIFY_EMAIL" | "IS_EMAIL_VERIFIED";
+    requestInit: RequestInit;
+    url: string;
+};
+
+export type InputType = RecipeConfig<PreAPIHookContext> & {
+    mode?: "OFF" | "REQUIRED";
+    disableDefaultImplementation?: boolean;
```

Comment:
You don't need `disableDefaultImplementation`

Reply:
Done